lib/dependencies/preview.ts

```diff
@@ -0,0 +1,4 @@
+import * as dependencies from "./dependencies";
```

Comment:
I think you can move all of these source files into the "lib" folder instead of the "lib/dependencies" folder, right?